src/harryPotterReader/Word.java

```diff
@@ -0,0 +1,23 @@
+package harryPotterReader;
+
+public class Word {
+	
+	private String word;
+	private int count = 1;
+	
+	public Word(String s) {
+		word = s;
+	}
+	
+	public void addCount() {
+		count += 1;
+	}
+
+	public String getWord() {
```

Comment:
GOOD AP HABIT THAT WOULD GET YOU POINTS ON AP

I, personally, think it is such a redundant thing, but it is probably just me.